apix-impl/src/test/java/eu/xenit/apix/tests/metadata/CreateContentWithMimetypeGuessUnitTest.java

```diff
@@ -28,42 +28,49 @@
 
     private static final Logger log = LoggerFactory.getLogger(CreateContentWithMimetypeGuessUnitTest.class);
 
-    @Test
-    public void createContentWithMimetypeGuess() throws UnsupportedEncodingException {
-        String fileName = "test";
+    @Ignore
```

Comment:
Why do you disable this test?

Reply:
Hm, unless I am missing something, can it be this whole file doesn't do anything?